crates/router/src/connector/payone/transformers.rs

```diff
@@ -1,227 +1,298 @@
+#[cfg(feature = "payouts")]
+#[derive(Debug, Serialize, Deserialize)]
+#[serde(rename_all = "camelCase")]
+pub struct OmnichannelPayoutSpecificInput {
+    payment_id: String,
 }
+#[cfg(feature = "payouts")]
+pub struct CardAndCardIssuer(Card, CardIssuer);
 
-impl<F> TryFrom<&PayoneRouterData<&types::RefundsRouterData<F>>> for PayoneRefundRequest {
-    type Error = error_stack::Report<errors::ConnectorError>;
+#[cfg(feature = "payouts")]
+impl TryFrom<PayoneRouterData<&types::PayoutsRouterData<api::PoFulfill>>>
+    for PayonePayoutFulfillRequest
+{
+    type Error = Error;
     fn try_from(
-        item: &PayoneRouterData<&types::RefundsRouterData<F>>,
+        item: PayoneRouterData<&types::PayoutsRouterData<api::PoFulfill>>,
     ) -> Result<Self, Self::Error> {
-        Ok(Self {
-            amount: item.amount.to_owned(),
-        })
+        let request = item.router_data.request.to_owned();
+        match request.payout_type.to_owned() {
+            storage_enums::PayoutType::Card => {
+                let amount_of_money: AmountOfMoney = AmountOfMoney {
+                    amount: item.router_data.request.amount,
+                    currency_code: item.router_data.request.destination_currency.to_string(),
+                };
+                let _card_issuer =
+                    CardAndCardIssuer::try_from(&item.router_data.get_payout_method_data()?)?;
+                let card = _card_issuer.0;
+                let card_issuer = _card_issuer.1;
+
+                let card_payout_method_specific_input: CardPayoutMethodSpecificInput =
+                    CardPayoutMethodSpecificInput {
+                        #[allow(clippy::as_conversions)]
+                        payment_product_id: Gateway::try_from(card_issuer)? as i32,
+                        card,
+                    };
+                Ok(Self {
+                    amount_of_money,
+                    card_payout_method_specific_input,
+                })
+            }
+            _ => Err(errors::ConnectorError::NotImplemented(
+                get_unimplemented_payment_method_error_message("Payone"),
+            ))?,
+        }
     }
 }
 
-// Type definition for Refund Response
+#[derive(Clone, Debug, Eq, Hash, PartialEq, Deserialize, Serialize)]
+pub enum Gateway {
+    Visa = 1,
+    MasterCard = 3,
+}
 
-#[allow(dead_code)]
-#[derive(Debug, Serialize, Default, Deserialize, Clone)]
-pub enum RefundStatus {
-    Succeeded,
-    Failed,
-    #[default]
-    Processing,
+impl TryFrom<CardIssuer> for Gateway {
+    type Error = error_stack::Report<errors::ConnectorError>;
+    fn try_from(issuer: CardIssuer) -> Result<Self, Self::Error> {
+        match issuer {
+            CardIssuer::Master => Ok(Self::MasterCard),
+            CardIssuer::Visa => Ok(Self::Visa),
+            _ => Err(errors::ConnectorError::NotImplemented(
+                get_unimplemented_payment_method_error_message("payone"),
+            )
+            .into()),
+        }
+    }
 }
```

Comment:
Why are we not covering all the card issuer networks?

Reply:
Those are not implemented